ironfish-cli/src/commands/wallet/mint.ts

```diff
@@ -142,42 +160,116 @@ export class Mint extends IronfishCommand {
     }
 
     let fee
-    if (flags.fee) {
-      fee = flags.fee
-    } else {
-      const input = await CliUx.ux.prompt(
-        `Enter the fee amount in $IRON (min: ${CurrencyUtils.renderIron(1n)})`,
-        {
-          default: CurrencyUtils.renderIron(1n),
-          required: true,
-        },
-      )
+    let rawTransactionResponse: string
 
-      fee = await parseIron(input, { largerThan: 0n, flagName: 'fee' }).catch((error: Error) =>
-        this.error(error.message),
-      )
+    try {
```

Comment:
this code crashes somewhere when I use this with this error

```
 ›   Error: Request failed (400) validation: fee is a required field
```

Reply:
im able to run it locally without any issue.